### PR TITLE
XW-1185 | Mercury Category Pages selenium tests selectors

### PR DESCRIPTION
**Pull request by Warkot**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1185
* https://github.com/Wikia/selenium-tests/pull/1118

## Description

Added few selectors for newly added selenium tests.
(f95e8c8 shows here, because I rebased once against `dev` not `XW-1185`.)

## Reviewers

@Wikia/x-wing

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: front/main/app/templates/components/category-sections.hbs [hunk 1]
```diff
@@ -5,5 +5,5 @@
 		loadBatch=attrs.loadBatch
 	}}
 {{else}}
-	{{i18n 'app.category-page-no-members'}}
+	<div class="category-page-no-members">{{i18n 'app.category-page-no-members'}}</div>
```
Comment (hakubo) [inline front/main/app/templates/components/category-sections.hbs]: should that be a state? category-page--no-mebmers
@vforge

File: front/main/app/templates/components/category-section.hbs [hunk 2]
```diff
@@ -7,7 +7,7 @@
 			radius=16
 			strokeWidth=2
 		}}
-		<button class="category-navigation__button" {{action 'loadBatch' id data.prevBatch 'previous'}}>
+		<button class="category-navigation__button previous-button" {{action 'loadBatch' id data.prevBatch 'previous'}}>
```
Comment (hakubo) [inline front/main/app/templates/components/category-section.hbs]: category-navigation__button category-navigation__button--previous
that sounds like a state or a variant
@vforge

File: front/main/app/templates/components/category-sections.hbs (hunk 1 above)
Comment (vforge) [inline front/main/app/templates/components/category-sections.hbs]: we discussed that offline and decided to proceed with data attributes `data-category-no-members`

File: front/main/app/templates/components/category-section.hbs (hunk 2 above)
Comment (vforge) [inline front/main/app/templates/components/category-section.hbs]: we discussed that offline and decided to proceed with data attributes: `data-prev` here

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**